src/ui/components/LedgerViewer.js

```diff
@@ -59,8 +61,7 @@ export const LedgerViewer = ({
   const {ledger} = useLedger();
   const classes = useStyles();
   const [allocation, setAllocation] = useState<Allocation | null>(null);
-  const [page, setPage] = useState(0);
-  const [rowsPerPage, setRowsPerPage] = React.useState(25);
+  const DATE_SORT = ["Date", (e) => e.ledgerTimestamp];
```

Comment:
nit: I advocate for named vs positional attributes. E.g.:
```suggestion
  const DATE_SORT = {
      name: "Date", 
      fn: (e) => e.ledgerTimestamp,
  };
```
then you can access them below like:
```
    sortName: DATE_SORT.name,
    sortOrder: SortOrders.DESC,
    sortFn: DATE_SORT.fn,
```

The reason being is it takes less cognitive load to remember positions (though presumably typing takes care of accidentally using the wrong one?) and it's more a descriptive when reading back the code.

Reply:
accepted